internal/downloader/download.go

```diff
@@ -67,7 +67,7 @@ func (d *Downloder) GetKubectlBinary(version semver.Version, destination string)
 		}
 	}
 
-	return d.download(downloadURL, destination, 0755)
+	return d.download(fmt.Sprintf("kubectl-%s%s", version, osexec.Ext), downloadURL, destination, 0755)
```

Comment:
The name of the kubectl binary changed recently, there's no more `-` separating `kubectl` from its version:
```suggestion
	return d.download(fmt.Sprintf("kubectl%s%s", version, osexec.Ext), downloadURL, destination, 0755)
```